### ISSUE TITLE
Building on Ubuntu 20.04

### ISSUE THREAD

**Issue by jackhumphries** [black-parrot/black-parrot-sim#1]
Hello,

My colleagues and I needed to run these additional commands to get BlackParrot to run on Ubuntu 20.04:

```
apt install pip verilator pkg-config uuid-dev default-jre tcl
pip install pytest
pip install pytest-pythonpath
```

Furthermore, I needed to create an alias for `cmake3` to `cmake`.

I didn't immediately see these commands in the instructions, so it may be helpful to add them.

**Comment by omasanori** [comment]
Could you describe when `pytest` packages are required?

**Comment by MaahAnvari** [comment]
How did you create an alias for cmake?
I am trying to make it but I got the cmake3: command not found.

**Comment by dpetrisko** [comment]
I believe this is resolved with the latest instructions (the included docker image is 20.04 and builds out of box)
